frontend/public/locales/en/alerts.json

```
@@ -18,5 +18,19 @@
       "email-valid": "Enter an email address in the correct format, such as name@example.com",
       "preferred-language-required": "Select preferred language of communication"
     }
+  },
+  "confirm": {
+    "page-title": "Submission Confirmation",
+    "breadcrumbs": {
+      "subscribe": "Confirm to subscribe to CDCP email alerts"
+    },
+    "no-preferred-language-on-file": "No preferred language on file",
+    "confirmation-information-text": "A confirmation code, required to finist the registration process, will be sent to clientsname@domain.com. This step must be completed within teh next 48 hours or by immediately entering the confirmation code below.",
```

```suggestion
    "confirmation-information-text": "A confirmation code, required to finish the registration process, will be sent to clientsname@domain.com. This step must be completed within the next 48 hours or by immediately entering the confirmation code below.",
```

Also, `clientsname@domain.com` shouldn't be hard-coded but rather taken from the input of the previous step (`/alerts/subscribe`).

---

Ive added a todo comment in the text, when we can retrieve the email address from the new source, i will modify this section to use it correctly

---

Sounds good. 

**Nitpick:** The typos still exist though